ci/scenarios/vmachine/perform_fio_test/main.py

```diff
@@ -121,11 +128,15 @@ def validate_fio_on_vdisks(amount_vdisks=AMOUNT_VDISKS, amount_to_write=AMOUNT_T
                                              "protocol `{3}` & diskname `{4}`"
                                              .format(image_path, storage_ip, edge_port, protocol, disk_name))
                 FioOnVDiskChecks.LOGGER.info("Converting image...")
-                client.run(["qemu-img", "convert", image_path, "openvstorage+{0}:{1}:{2}/{3}"
-                           .format(protocol, storage_ip, edge_port, disk_name)])
+                edge_info = {'port': edge_port,
+                             'protocol': protocol,
+                             'ip': storage_ip,
```

Comment:
Unnecessary comma. Also append the } to here because you are starting with an inline {